processors/config/kalSimpTuple_cfg.py

```diff
@@ -107,11 +119,17 @@
 trackgbl.parameters["bfield"] = bfield[str(options.year)]
 
 #ECalData
-ecal.parameters["debug"] = 0
-ecal.parameters["hitCollLcio"] = 'EcalCalHits'
-ecal.parameters["hitCollRoot"] = ''#'RecoEcalHits'
-ecal.parameters["clusCollLcio"] = "EcalClustersCorr"
-ecal.parameters["clusCollRoot"] = "RecoEcalClusters"
+ecal_gbl.parameters["debug"] = 0
+ecal_gbl.parameters["hitCollLcio"] = 'EcalCalHits'
+ecal_gbl.parameters["hitCollRoot"] = 'RecoEcalHits'
+ecal_gbl.parameters["clusCollLcio"] = "EcalClustersCorr"
+ecal_gbl.parameters["clusCollRoot"] = "RecoEcalClusters"
+
```

Comment:
What is the difference between ecal_gbl and ecal_kf? They look the same to me.

Reply:
ah yeah, they were diff when I was running both recos at the same time. I will remove that now